### ISSUE TITLE
ComboBox requires tab twice to navigate over

### ISSUE THREAD

**Issue by JoeGames** [firstfloorsoftware/mui#99]
The issue is that all ComboBox controls require hitting tab twice to cycle over.

I think the cause is the ToggleButton is grabbing focus separately from the control itself.

**Comment by heraclesaraujo** [comment]
Yes, that is the cause... I had to make some changes on the default templates to get around it
